content/en/docs/contribute/start.md

```diff
@@ -294,15 +294,31 @@ contribution guide.
 
 ## Review docs pull requests
 
-People who are not yet approvers or reviewers can still review pull requests. 
-The reviews are not considered "binding", which means that your review alone 
-won't cause a pull request to be merged. However, it can still be helpful. Even 
-if you don't leave any review comments, you can get a sense of pull request 
-conventions and etiquette and get used to the workflow. Take a look at the 
-[Content](/docs/contribute/style/content-guide/) and 
-[Style](/docs/contribute/style/style-guide/) guides before you review so you 
+People who are new to documentation can still review pull requests. You can
+learn the code base and build trust with your fellow contributors. English docs
+are the authoritative source for content. We communicate in English during
+weekly meetings and in community announcements. Contributors' English skills
+vary, so use simple and direct language in your reviews. Effective reviews focus
+on both small details and a change's potential impact.
+
+The reviews are not considered "binding", which means that your review alone
+won't cause a pull request to be merged. However, it can still be helpful. Even
+if you don't leave any review comments, you can get a sense of pull request
+conventions and etiquette and get used to the workflow. Familiarize yourself with the
+[content guide](/docs/contribute/style/content-guide/) and
+[style guide](/docs/contribute/style/style-guide/) before reviewing so you
 get an idea of what the content should contain and how it should look.
 
+### Best practices
```

Comment:
```suggestion
### Best practices
```
In various different places, I've seen people write “good practice” rather than “best practice”.